src/hard-tabs-transform.coffee

```diff
@@ -1,8 +1,10 @@
+require("es6-shim")
```

Comment:
Good point, I'll require it in spec-helper only. I don't know if it's an option right now, but starting from v0.11 we can simply use `--harmony_strings` flag.